src/pkg/credentials/m365.go

```diff
@@ -12,6 +14,12 @@ const (
 	AzureClientSecret = "AZURE_CLIENT_SECRET"
 )
 
+var (
+	AzureClientTenantFV string
```

Comment:
Same thing here: flag values should not exist below the cli layer.  These declarations should get moved to `cli/utils.go` and passed into this func.

Reply:
As mentioned above. Its because of cyclic dependency.